package.json

```diff
@@ -21,13 +21,16 @@
     "coap": "^0.23.1",
     "core-js": "^2.6.5",
     "electron-json-storage": "^4.1.8",
+    "i": "^0.3.6",
```

Comment:
Could you explain the added dependencies?

Reply:
And the removed vuex-persistfile

Reply:
Just removed "i" package, I must've accidentally installed it and hadn't noticed. `vuex-persistfile` I removed since it is not used anywhere (`vuex-electron` is used for persisting to files)